Manifests/ManagedPreferencesApplications/edu.ncsu.confboard.plist

```diff
@@ -0,0 +1,290 @@
+			<key>pfm_name</key>
+			<string>baseURL</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>baseURL</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_default</key>
+			<string>https://storage.googleapis.com/confboard</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Base URL is a JSON file or feed not text list of URIs. </string>
+			<key>pfm_documentation_url</key>
+			<string>https://go.ncsu.edu/confboard#h.as2mra9pviie</string>
+			<key>pfm_name</key>
+			<string>shouldUseJSON</string>
+			<key>pfm_title</key>
+			<string>Use ConfBoard JSON data file</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Show name of device on screen</string>
+			<key>pfm_documentation_url</key>
+			<string>https://go.ncsu.edu/confboard#h.8bfb1vc4d0zo</string>
+			<key>pfm_name</key>
+			<string>shouldDisplayName</string>
+			<key>pfm_title</key>
+			<string>Display Device Name</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>String to be added after the baseURL.  Using images.txt this adds after the baseURL and before images.txt.  Using JSON this adds after the baseURL. Can be used to specify a folder on web server with different images.txt or as ? parameter string for JSON feed. </string>
+			<key>pfm_documentation_url</key>
+			<string>https://go.ncsu.edu/confboard#h.8tq6r5jg8sne</string>
+			<key>pfm_name</key>
+			<string>groupLabel</string>
+			<key>pfm_require</key>
+			<string>never</string>
+			<key>pfm_title</key>
+			<string>Group Label</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_default</key>
+			<string></string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<integer>720</integer>
+			<key>pfm_description</key>
+			<string>Time between reloading the list of image links</string>
+			<key>pfm_documentation_url</key>
+			<string>https://go.ncsu.edu/confboard#h.97mek2dcu8c</string>
+			<key>pfm_name</key>
+			<string>reloadContentTimer</string>
+			<key>pfm_require</key>
+			<string>always</string>
+	
+			<key>pfm_range_list</key>
+			<array>
+				<integer>30</integer>
+				<integer>60</integer>
+				<integer>120</integer>
+				<integer>240</integer>
+				<integer>480</integer>
+				<integer>720</integer>
+				<integer>1440</integer>
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>30 Minutes</string>
+				<string>1 Hour</string>
+				<string>2 Hours</string>
+				<string>4 Hours</string>
+				<string>8 Hours</string>
+				<string>12 Hours</string>
+				<string>24 Hours</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Reload Image List Time</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+			<key>pfm_value_unit</key>
+			<string>minutes</string>
```

Comment:
If you specify a `pfm_value_unit` we don't want to then also specify the unit in the `pfm_range_list_titles`.  I think the string with the time values make sense, and therefore should remove the `pfm_value_unit` key altogether.